quotes/whydnxx.yaml

```diff
@@ -0,0 +1,5 @@
+username: whydn.dev
```

Comment:
Can we have same github username, just like filename and your github username. And also the author name should your real name or github username. Thank you @whydnxx